Prose/ProseLib/Sources/App/AppReducer.swift

```diff
@@ -30,21 +30,42 @@ public let appReducer: Reducer<
     ),
     Reducer { state, action, environment in
         switch action {
+        case .onAppear:
+            guard case .auth = state.route else { return .none }
+
+            guard let jid = environment.userDefaults.loadCurrentAccount() else { return .none }
+
+            do {
+                guard let password = try environment.credentials.loadCredentials(jid) else { return .none }
+                return Effect(value: .auth(.didLogIn(jid: jid, password: password)))
+            } catch {
+                // TODO: Handle this error, user might have denied access to the credentials
+                print("Error when loading credentials: \(error.localizedDescription)")
```

Comment:
There's nothing to do for us here. We could log the error for informational purposes but the user made their decision or we have an error that we can do nothing about other than show the login screen again. No need to handle this error.

Reply:
Then we still need to handle the error, by showing the login screen.

Reply:
The route is already set to `.auth` in this code path. If you don't crash the login form will be shown.

Reply:
Oh yes, it is on this branch, I just remembered [changing this in the next PR](https://github.com/prose-im/prose-app-macos/pull/38/commits/1592ec7cd70284823401eed611b64c471987b488#diff-7e699972e390677c42757230305df7bd9805efc806d35b0e57f362e3e99cd06aR21).

Do you mind if I leave it like this?